index.html

```diff
@@ -22,16 +22,21 @@
       <nav class="mobile-menu">
         <h1 class="logo">My Logo</h1>
         <img class="navMenu" src="images/hamburger.png" id="trigger" alt="menu icon" />
+        <ul id="info-text">
+        <li class="text-1">Portfolio</li>
+         <li class="text-1">About</li>
+        <li class="text-1">Contact</li>
+        </ul>
       </nav>
     </header>
       <section id="landing">
        
-        <div class="hello"><h2>I’m Francis, Glad to see you!</h2></div>
+        <div class="hello"><h2>I’m Francis<br> Glad to see you!</h2></div>
```

Comment:
It's almost done, but as you know it's required to follow the exact Figma design, so, please make sure to use the same font weight for the title and arrange your social media icons properly.

![image](https://user-images.githubusercontent.com/34779928/194647016-b0f5bb65-df02-4cc6-953c-6e78dd8bf565.png)